classes/class-editor.php

```diff
@@ -111,14 +111,16 @@ function() {
 			add_action( 'wp_print_footer_scripts', array( '_WP_Editors', 'print_default_editor_scripts' ), 45 );
 		}
 
-		// Gutenberg styles
-		wp_enqueue_style( 'wp-edit-post' );
-		wp_enqueue_style( 'wp-format-library' );
+		// Optionally skip loading the editor styles.
+		if ( ! defined( 'NO_EDITOR_STYLES' ) ) {
```

Comment:
Maybe we should use a filter instead? And this name is too generic, it will be defined on the entire host site. It should be prefixed with `BLOCKS_EVERYWHERE_`.